### PR TITLE
adds checks for the new radio buttons and amends commenting

### PR DESCRIPTION
**Pull request by omehes**
we have 2 new radio buttons in the cops modal (pdf and distribution preview). added tests for these and amended some commentary in the code.

### CODE REVIEW

**Comment by omehes** [comment]
The failed test is nothing I touched, not related to my changes.

**Comment by rnathuji** [comment]
@omehes - Just FYI, that failure is something we're aware of and looking into (see [cnx 1168](https://github.com/openstax/cnx/issues/1168)).

**Comment by omehes** [comment]
@rnathuji thanks for letting me know

**Review by tomjw64 — APPROVED** [review]

File: backend/app/tests/ui/pages/home.py [hunk 1]
```diff
@@ -90,6 +94,14 @@ def click_create_button(self):
             self.wait.until(lambda _: self.page.create_job_modal_is_open)
             sleep(2)
 
+        @property
+        def distribution_preview_radio_button(self):
+            return self.find_element(*self._modal_preview_radio_button_locator)
+
+        def click_distribution_preview_radio_button(self):
+            self.distribution_preview_radio_button.click()
+            # self.wait.until(lambda _: self.page.create_job_modal_is_open)
```
Comment (tomjw64) [inline backend/app/tests/ui/pages/home.py:103]: is this comment still needed?

Comment (omehes) [reply backend/app/tests/ui/pages/home.py:103]: @tomjw64 aaahhhh... I just didn't see it there. I will delete it and re-push the changes for review

Comment (omehes) [reply backend/app/tests/ui/pages/home.py]: @tomjw64 done. removed the comment.

Comment (tomjw64) [reply backend/app/tests/ui/pages/home.py]: my approving review stands, you should be able to merge

**Pull request merged**